feature-libs/cart/import-export/components/import-to-cart/import-to-cart.service.ts

```diff
@@ -55,12 +63,12 @@ export class ImportToCartService {
               });
               this.savedCartService.loadSavedCarts();
             }),
-            // TODO: #13392
-            // delayWhen((cartId: string) =>
-            //   this.savedCartService
-            //     .isStable(cartId)
-            //     .pipe(filter((stable: boolean) => stable))
-            // ),
+            observeOn(queueScheduler),
```

Comment:
without it, the following `delayWhen()` would use the "old" `loading` value (false)